src/main/java/org/junitpioneer/internal/PioneerAnnotationUtils.java

```diff
@@ -204,18 +207,52 @@ private static boolean isContainerAnnotationOf(Annotation potentialContainer, Cl
 					.orElse(Collections.emptyList());
 	}
 
+	private static <A extends Annotation> List<A> findOnClass(Class<?> element, Class<A> annotationType,
+			boolean findRepeated, boolean findAllEnclosing) {
+		if (element == null || element == Object.class)
+			return Collections.emptyList();
+		if (findRepeated)
```

Comment:
this matches with this, cause you  search here for multiple usages of a repeatable annotation. If you already know that it was reapeated, then you would not need this find.

Reply:
Renaming this would make it inconsistent with the rest of the class, though 🤔